pages/docs/adapters/calculix/adapter-calculix-pastix-build.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Building the Calculix adapter with PaStiX (and CalculiX 2.17)
+permalink: adapter-calculix-pastix-build.html
+keywords: adapter, calculix, pastix
+summary: "Since version 2.17, CalculiX can be built with the PaStiX library to increase performance with CUDA. Building the preCICE adapter
+is somehow harder with this version. This page gives a detailed walkthrough to build the modified adapter."
+---
+
+## Overview
+
+To build the preCICE adapter for CalculiX with PaStiX, it is first necessary to build PaStiX. A manual build is necessary for most steps, as specific compilation flags must be used for the dependencies: pre-built packages cannot be used. In particular, PaStiX (and CalculiX) must be compiled with flags to use 8-byte integers. On this page, we provide a step-by-step building guide of the adapter, tested on Ubuntu 20.04.
+This should work with some modifications on other systems. You may also need to tweak this depending on your needs, e.g. if you want to build some dependencies yourself instead of using a package manager.
+
+## Required packages
+
+Make sure you have a working installation of preCICE. Also run these installation commands, (after a call to `sudo apt update` and `sudo apt upgrade`) :
+`sudo apt install build-essential cmake git gfortran flex bison zlib1g-dev nvidia-cuda-toolkit-gcc libspooles-dev libyaml-cpp-dev`
+
+## Downloading CalculiX source
+
+This guide assumes Calculix's source code is in the user's home folder `/home/user_name`, with the alias `~`. If you don't want to follow this convention, you may have to adapt slightly the instructions below. Donwload can be done on command line:
+
+```bash
+    cd ~ && wget http://www.dhondt.de/ccx_2.17.src.tar.bz2
+    bunzip2 ccx_2.17.src.tar.bz2
+    tar -xvf ccx_2.17.src.tar
```

Comment:
See also https://github.com/precice/precice.github.io/pull/81 (also for any other hard-coded CalculiX version here)

Reply:
Yes, I agreee with both suggestions (notes & CCX version).
Until the notes update is merged I should probably add it as plain text then?

Also PaStiX is only available since 2.17, so using the website variable seems inappropriate as long as it refers to 2.16

Reply:
The notes update has been merged now.